{"created": 1741486551.061942, "duration": 0.6284952163696289, "exitcode": 0, "root": "/Users/joan/VSProjects/pyERP/tests", "environment": {}, "summary": {"passed": 8, "total": 8, "collected": 8}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": "api", "type": "Package"}]}, {"nodeid": "api/test_direct_api/test_client.py::TestLegacyERPClient", "outcome": "passed", "result": [{"nodeid": "api/test_direct_api/test_client.py::TestLegacyERPClient::test_check_connection", "type": "TestCaseFunction", "lineno": 106}, {"nodeid": "api/test_direct_api/test_client.py::TestLegacyERPClient::test_fetch_table_error_handling", "type": "TestCaseFunction", "lineno": 97}, {"nodeid": "api/test_direct_api/test_client.py::TestLegacyERPClient::test_fetch_table_pagination", "type": "TestCaseFunction", "lineno": 71}, {"nodeid": "api/test_direct_api/test_client.py::TestLegacyERPClient::test_fetch_table_success", "type": "TestCaseFunction", "lineno": 50}]}, {"nodeid": "api/test_direct_api/test_client.py", "outcome": "passed", "result": [{"nodeid": "api/test_direct_api/test_client.py::TestLegacyERPClient", "type": "UnitTestCase"}]}, {"nodeid": "api/test_direct_api/test_exceptions.py::TestLegacyERPExceptions", "outcome": "passed", "result": [{"nodeid": "api/test_direct_api/test_exceptions.py::TestLegacyERPExceptions::test_authentication_error", "type": "TestCaseFunction", "lineno": 31}, {"nodeid": "api/test_direct_api/test_exceptions.py::TestLegacyERPExceptions::test_connection_error", "type": "TestCaseFunction", "lineno": 25}, {"nodeid": "api/test_direct_api/test_exceptions.py::TestLegacyERPExceptions::test_legacy_erp_error", "type": "TestCaseFunction", "lineno": 20}, {"nodeid": "api/test_direct_api/test_exceptions.py::TestLegacyERPExceptions::test_response_error", "type": "TestCaseFunction", "lineno": 37}]}, {"nodeid": "api/test_direct_api/test_exceptions.py", "outcome": "passed", "result": [{"nodeid": "api/test_direct_api/test_exceptions.py::TestLegacyERPExceptions", "type": "UnitTestCase"}]}, {"nodeid": "api/test_direct_api", "outcome": "passed", "result": [{"nodeid": "api/test_direct_api/test_client.py", "type": "Module"}, {"nodeid": "api/test_direct_api/test_exceptions.py", "type": "Module"}]}, {"nodeid": "api", "outcome": "passed", "result": [{"nodeid": "api/test_direct_api", "type": "Package"}]}], "tests": [{"nodeid": "api/test_direct_api/test_client.py::TestLegacyERPClient::test_check_connection", "lineno": 106, "outcome": "passed", "keywords": ["test_check_connection", "TestLegacyERPClient", "test_client.py", "test_direct_api", "api", "tests", ""], "setup": {"duration": 0.12475833401549608, "outcome": "passed"}, "call": {"duration": 0.0019707499886862934, "outcome": "passed", "stderr": "Checking connection to legacy ERP API\nChecking connection to legacy ERP API\nConnection check failed: Connection failed\n", "log": [{"name": "pyerp.external_api.legacy_erp.client", "msg": "Checking connection to legacy ERP API", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/Users/joan/VSProjects/pyERP/pyerp/external_api/legacy_erp/client.py", "filename": "client.py", "module": "client", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 52, "funcName": "check_connection", "created": 1741486550.632164, "msecs": 632.0, "relativeCreated": 11518.054962158203, "thread": 8590017088, "threadName": "MainThread", "processName": "MainProcess", "process": 97478, "taskName": null}, {"name": "pyerp.external_api.legacy_erp.client", "msg": "Checking connection to legacy ERP API", "args": null, "levelname": "INFO", "levelno": 20, "pathname": "/Users/joan/VSProjects/pyERP/pyerp/external_api/legacy_erp/client.py", "filename": "client.py", "module": "client", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 52, "funcName": "check_connection", "created": 1741486550.632607, "msecs": 632.0, "relativeCreated": 11518.497943878174, "thread": 8590017088, "threadName": "MainThread", "processName": "MainProcess", "process": 97478, "taskName": null}, {"name": "pyerp.external_api.legacy_erp.client", "msg": "Connection check failed: Connection failed", "args": null, "levelname": "ERROR", "levelno": 40, "pathname": "/Users/joan/VSProjects/pyERP/pyerp/external_api/legacy_erp/client.py", "filename": "client.py", "module": "client", "exc_info": null, "exc_text": null, "stack_info": null, "lineno": 55, "funcName": "check_connection", "created": 1741486550.632732, "msecs": 632.0, "relativeCreated": 11518.622875213623, "thread": 8590017088, "threadName": "MainThread", "processName": "MainProcess", "process": 97478, "taskName": null}]}, "teardown": {"duration": 0.0006692089955322444, "outcome": "passed"}}, {"nodeid": "api/test_direct_api/test_client.py::TestLegacyERPClient::test_fetch_table_error_handling", "lineno": 97, "outcome": "passed", "keywords": ["test_fetch_table_error_handling", "__wrapped__", "patchings", "TestLegacyERPClient", "test_client.py", "test_direct_api", "api", "tests", ""], "setup": {"duration": 0.0009185829840134829, "outcome": "passed"}, "call": {"duration": 0.005779833009000868, "outcome": "passed"}, "teardown": {"duration": 0.00028995799948461354, "outcome": "passed"}}, {"nodeid": "api/test_direct_api/test_client.py::TestLegacyERPClient::test_fetch_table_pagination", "lineno": 71, "outcome": "passed", "keywords": ["test_fetch_table_pagination", "__wrapped__", "patchings", "TestLegacyERPClient", "test_client.py", "test_direct_api", "api", "tests", ""], "setup": {"duration": 0.000832874997286126, "outcome": "passed"}, "call": {"duration": 0.016480666992720217, "outcome": "passed"}, "teardown": {"duration": 0.0003052910033147782, "outcome": "passed"}}, {"nodeid": "api/test_direct_api/test_client.py::TestLegacyERPClient::test_fetch_table_success", "lineno": 50, "outcome": "passed", "keywords": ["test_fetch_table_success", "__wrapped__", "patchings", "TestLegacyERPClient", "test_client.py", "test_direct_api", "api", "tests", ""], "setup": {"duration": 0.0008332919969689101, "outcome": "passed"}, "call": {"duration": 0.0012091250100638717, "outcome": "passed"}, "teardown": {"duration": 0.00038949999725446105, "outcome": "passed"}}, {"nodeid": "api/test_direct_api/test_exceptions.py::TestLegacyERPExceptions::test_authentication_error", "lineno": 31, "outcome": "passed", "keywords": ["test_authentication_error", "TestLegacyERPExceptions", "test_exceptions.py", "test_direct_api", "api", "tests", ""], "setup": {"duration": 0.0009237919875886291, "outcome": "passed"}, "call": {"duration": 0.0002442499971948564, "outcome": "passed"}, "teardown": {"duration": 0.0002853330224752426, "outcome": "passed"}}, {"nodeid": "api/test_direct_api/test_exceptions.py::TestLegacyERPExceptions::test_connection_error", "lineno": 25, "outcome": "passed", "keywords": ["test_connection_error", "TestLegacyERPExceptions", "test_exceptions.py", "test_direct_api", "api", "tests", ""], "setup": {"duration": 0.0007727500051259995, "outcome": "passed"}, "call": {"duration": 0.00022100002388469875, "outcome": "passed"}, "teardown": {"duration": 0.00024058297276496887, "outcome": "passed"}}, {"nodeid": "api/test_direct_api/test_exceptions.py::TestLegacyERPExceptions::test_legacy_erp_error", "lineno": 20, "outcome": "passed", "keywords": ["test_legacy_erp_error", "TestLegacyERPExceptions", "test_exceptions.py", "test_direct_api", "api", "tests", ""], "setup": {"duration": 0.0007425830117426813, "outcome": "passed"}, "call": {"duration": 0.0002082499850075692, "outcome": "passed"}, "teardown": {"duration": 0.00023395800963044167, "outcome": "passed"}}, {"nodeid": "api/test_direct_api/test_exceptions.py::TestLegacyERPExceptions::test_response_error", "lineno": 37, "outcome": "passed", "keywords": ["test_response_error", "TestLegacyERPExceptions", "test_exceptions.py", "test_direct_api", "api", "tests", ""], "setup": {"duration": 0.0007345830090343952, "outcome": "passed"}, "call": {"duration": 0.0002634999982547015, "outcome": "passed"}, "teardown": {"duration": 0.001132250006776303, "outcome": "passed"}}], "warnings": [{"message": "Unknown config option: env\n", "category": "PytestConfigWarning", "when": "config", "filename": "/Users/joan/VSProjects/pyERP/venv/lib/python3.12/site-packages/_pytest/config/__init__.py", "lineno": 1396}]}